public/locale/en.json

```
@@ -207,6 +207,12 @@
   "SCHEDULE_AVAILABILITY_TYPE__open": "Open",
   "SCHEDULE_EXCEPTION_TYPE__MODIFY_SCHEDULE": "Modify Schedule",
   "SCHEDULE_EXCEPTION_TYPE__UNAVAILABLE": "Unavailable",
+  "SESSION_EXPIRED_BREATHE": "Breathe",
+  "SESSION_EXPIRED_BREATH_IN": "in",
+  "SESSION_EXPIRED_BREATH_OUT": "out",
+  "SESSION_EXPIRED_LOGIN_AGAIN": "log in again",
```

the `ENUM_NAME__ENUM_VALUE` is used only for enum values. simply declare others like this:

```suggestion
  "breathe": "Breathe",
  "in": "in",
  "out": "out",
  "log_in_again": "log in again",
```